### PR TITLE
LPS-131793 Added docs&media as an associated content type for OOTB internal vocabularies by default

### PR DESCRIPTION
**Pull request by YurenaC**
As Documents and Media were added to the Content Dashboard, Internal Vocabularies can be used for auditing content so, docs&media should be an associated content type for OOTB internal vocabularies

### CODE REVIEW

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java [hunk 1]
```diff
@@ -56,15 +57,15 @@ public void portalInstanceRegistered(Company company) throws Exception {
 		_addAssetVocabulary(
 			company, "audience",
 			AssetVocabularyConstants.VISIBILITY_TYPE_INTERNAL,
-			JournalArticle.class);
+			JournalArticle.class, DLFileEntry.class);
```
Comment (4lejandrito) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java:60]: Sort these alphabetically (DL... goes before Journal...).

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java [hunk 2]
```diff
@@ -56,15 +57,15 @@ public void portalInstanceRegistered(Company company) throws Exception {
...
 		_addAssetVocabulary(
 			company, "stage", AssetVocabularyConstants.VISIBILITY_TYPE_INTERNAL,
-			JournalArticle.class);
+			JournalArticle.class, DLFileEntry.class);
 	}
 
 	private void _addAssetVocabulary(
 			Company company, String name, int visibilityType,
-			Class<?> assetVocabularyTypeClass)
+			Class<?>... assetVocabularyTypeClass)
```
Comment (4lejandrito) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java:68]: Rename this to `assetVocabularyTypeClasses`

**Review by 4lejandrito — CHANGES_REQUESTED** [review]

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java (hunk 1 above)
Comment (YurenaC) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/instance/lifecycle/AddDefaultAssetVocabulariesPortalInstanceLifecycleListener.java:60]: The associated asset types are shown in the same order as they are introduced, so to keep it as we expected (first web content and then documents) this is the way of doing it :)

**Comment by YurenaC** [comment]
ci:forward

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:forward

**Review by 4lejandrito — APPROVED** [review]

**Comment by YurenaC** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by YurenaC** [comment]
ci:forward

**Pull request closed without merging**